README.md

```diff
@@ -9,6 +9,11 @@ The JAXB framework enables developers to perform the following operations:
 
 JAXB gives Java developers an efficient and standard way of mapping between XML and Java code. Java developers using JAXB are more productive because they can write less code themselves and do not have to be experts in XML. JAXB makes it easier for developers to extend their applications with XML and Web Services technologies.
 
+# Codemodel
+Codemodel is an independent library for java code generation, which is released separately and integrated in JAXB. 
+Besides JAXB it is used in [istack-commons-project](https://github.com/bravehorsie/jaxb-istack-commons), 
```

Comment:
you don't want to point to your fork here

Reply:
:) ty